<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArabicTranslateClass</class>
 <widget class="QMainWindow" name="ArabicTranslateClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>1115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arabic Translation</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <widget class="QGroupBox" name="control_group">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>381</width>
      <height>941</height>
     </rect>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <widget class="QPushButton" name="pdf">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>390</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>PDF </string>
     </property>
    </widget>
    <widget class="QPushButton" name="view">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>470</y>
       <width>91</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>View</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>61</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Types</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_types">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>96</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="data_base_operations">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>361</width>
       <height>241</height>
      </rect>
     </property>
     <property name="title">
      <string>Data Base Operations</string>
     </property>
     <widget class="QPushButton" name="db_add">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="search_num">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>201</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="db_retrieve">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>80</y>
        <width>112</width>
        <height>34</height>
       </rect>
      </property>
      <property name="text">
       <string>Retrieve</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="data_view" native="true">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>981</width>
      <height>921</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1390</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="file_open"/>
    <addaction name="separator"/>
    <addaction name="file_save"/>
    <addaction name="separator"/>
    <addaction name="file_exit"/>
   </widget>
   <addaction name="menu_file"/>
  </widget>
  <widget class="QToolBar" name="tool_bar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="status_bar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>50</height>
    </size>
   </property>
  </widget>
  <action name="file_exit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="file_open">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="file_save">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources/mvesfm.qrc"/>
 </resources>
 <connections/>
</ui>
